mulesoft/docs-site-ui
# [W-11071448] doc site nav update helmet

ref: W-11071448

# Enhancements

Add an additional Salesforce banner (called "helmet") in the header.

Full screen size
![Screen Shot 2022-05-05 at 11 15 21 AM](https://user-images.githubusercontent.com/10934908/166989259-9b16893f-18a5-41f6-9558-1c08a3412d19.png)

Mobile/small screen size
![Screen Shot 2022-05-05 at 11 15 30 AM](https://user-images.githubusercontent.com/10934908/166989307-c3ce7df1-a0ea-4b59-af61-561f7c25e9b7.png)

Mobile/small screen size menu
![Screen Shot 2022-05-05 at 11 15 36 AM](https://user-images.githubusercontent.com/10934908/166989332-c65094c0-184a-4a8b-944e-a8608bf46ecc.png)

# How to validate

1. Open terminal, go to this repo, switch to this branch, pull the latest code
2. Run `gulp preview` and open localhost:5252 in the web browser (Chrome, Firefox, Safari)
3. Validate that the Salesforce "helmet" is available at the top of the banner
4. Validate that no header elements are overlapping
5. Decrease the screen size to small/mobile. Validate that the helmet is still at the top of the page, and no leader elements are overlapping
6. Scroll down on the page, validate that the helmet is stickied (aka doesn't disappear) at the top of the page
7. Expand the menu, the menu should be scrollable and you can see all the content. The helmet should still be visible and stickied at the top
